mlir/include/mlir/Dialect/Rock/IR/RockAttrDefs.td

```diff
@@ -222,15 +222,14 @@ def Rock_GeneralGemmParamsAttr : Rock_Attr<"GeneralGemmParams", [RockTuningParam
   );
 
   let extraClassDeclaration = [{
-    void getPerfConfigStr(std::string &perfStr){
-      const Twine paramsConcat =
+      std::string getPerfConfigStr(){
```

Comment:
I think the reference parameters were used to allow for populating the perf config repeatedly without reallocation. Perhaps changing from `std::string &` to `SmallVectorImpl<char> &` would also be worth considering.

In any case, @jungpark-mlir .

Reply:
(But if we don't hear anything soon I'm willing to approve)

Reply:
Changed it back to something similar to the original, but with all the Twines inside one set of parens and fed to a .str() call.  Apparently they manage to deallocate themselves between the assignment and the use in the original code.